lib/virt_autotest/utils.pm

```diff
@@ -116,6 +116,13 @@ sub is_kvm_host {
     return check_var("SYSTEM_ROLE", "kvm") || check_var("HOST_HYPERVISOR", "kvm") || check_var("REGRESSION", "qemu-hypervisor");
 }
 
+#retrun 1 if libvirt 9.1+ is running
+sub is_libvirt91 {
```

Comment:
I think you can put your libvirt subroutine in ```lib/services/libvirtd.pm``` and make it better.

Reply:
I looked at this file. it has not been used by our squard. All the libvirt functions created or used by us are located in lib/virt_autotest/* or tests/virt_autotests/virt_utils.pm. If we would consider of using lib/services/libvirtd.pm, we'd better move all libvirt related functions to it. It doesn't feel very suitable to only include this function.